### ISSUE TITLE
Bug 8780741: A11y_OneNoe Clipper_Verify accessibility with Change Clipper Mode_Region Button_Keyboard: On invoking the back button on the region dialog box, Keyboard focus is not falling back on the region button.

### ISSUE THREAD

**Issue by aanchalbhansali** [OneNoteDev/WebClipper#606]
Environment Details: 
Application Name: OneNote Clipper-Web
OS: Windows 11 OS build 26058.1000
Edge Dev Version: Version 123.0.2400.1 (Official build) dev (64-bit)
AT's: Keyboard
Pre-Requisites:
Open the URL: [OneNote Web Clipper Installation](https://www.onenote.com/clipper)
Invoke the 'Get OneNote web Clipper for Microsoft Edge' button.
Edge Add-ons page will appear. Invoke the Get button.
OneNote Clipper will be added in edge extension section. 
Expand the Extension button and turn on the OneNote Clipper Show in Toolbar button.
OneNote Clipper will appear beside of the search field.
Now invoke the OneNote clipper button.
Sign in with V-id.
Repro Steps:
Open the [Ankylosaurus - Wikipedia](https://en.wikipedia.org/wiki/Ankylosaurus)
Now invoke the OneNote clipper button.
OneNote Clipper dialog box will appear.
Tab to navigate to the Region button and invoke it. Region dialog box will appear.
Invoke the back button and observe the issue.
Actual Result:
On invoking the back button on the region dialog box, Keyboard focus is not falling back on the region button.
Expected Result:
On invoking the back button on the region dialog box, the Keyboard focus should be falling back on the region button.
User Impact:
Keyboard users might be confused, if there is no keyboard focus is not falling back to the related control.
MAS Reference:
[MAS 2.4.3 – Focus Order](https://aka.ms/MAS2.4.3)
